packages/primitives/tooltip/src/tooltip-root.directive.ts

```diff
@@ -70,7 +70,7 @@ export class RdxTooltipRootDirective implements OnInit {
     /**
      * The controlled open state of the tooltip. Must be used in conjunction with onOpenChange.
      */
-    readonly open = input<boolean | undefined, BooleanInput>(false, { transform: booleanAttribute });
+    readonly open = input<boolean | undefined, BooleanInput>(void 0, { transform: booleanAttribute });
```

Comment:
this fix is relevant, all the rest comes from PR #232 - that base PR must get merged before this one